app/controllers/comments_controller.rb

```diff
@@ -28,6 +28,13 @@ def update
     end
   end
 
+  def destroy
+    comment = Comment.find(params[:id])
```

Comment:
`comment = Comment.find_by!(giveaway_id: params[:giveaway_id], id: params[:id])`

Reply:
Thanks @khadzhinov, I guess this is the better way beacuse find_by! raises ActiveRecord::RecordNotFound error?

Reply:
Exactly!